src/v/cloud_storage/remote.cc

```diff
@@ -191,6 +191,9 @@ int remote::delete_objects_max_keys() const {
     case model::cloud_storage_backend::azure:
         // Will be supported once azurite supports batch blob delete
         [[fallthrough]];
+    case model::cloud_storage_backend::oracle:
```

Comment:
OCI claims to support bulk delete https://docs.oracle.com/en-us/iaas/api/#/en/s3objectstorage/20160918/Object/BulkDelete

At first glance it seems to match S3 API. Why are you explicitly disabling it?

Reply:
Argh. Missed this in the API docs. Thank you for the call-out!